src/bootstrap/setup.rs

```diff
@@ -372,34 +400,77 @@ pushing your code to ensure your code is up to par. If you decide later that thi
 undesirable, simply delete the `pre-push` file from .git/hooks."
     );
 
-    let should_install = loop {
+    loop {
```

Comment:
Can you factor this out into a function so it isn't duplicated between both prompts?